src/AbstractQuery.php

```diff
@@ -491,22 +473,6 @@ protected function buildOrderBy()
      */
     protected function buildLimit()
     {
-        $clause = '';
```

Comment:
as it says in the comment: it is a template method https://en.wikipedia.org/wiki/Template_method_pattern "one or more algorithm steps can be overridden by subclasses to allow differing behaviors while ensuring that the overarching algorithm is still followed"
The buildLimit() is here to hold its place in the algorithm for a derived method to do something useful with.